docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
-{{% /alert %}}
+**Output:**
 
-## Next steps
+```bash
+NAME: cert-manager
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
 
-* If you're a developer, and you'd like to get started coding using CQL or Stargate, see the [Quickstart for developers]({{< relref "/quickstarts/developer" >}}).
-* If you're a Site Reliability Engineer, and you'd like to explore the K8ssandra administration environment including monitoring and maintenance utilities, see the [Quickstart for Site Reliability Engineers]({{< relref "/quickstarts/site-reliability-engineer" >}}).
+Then enter:
+
+```bash
+kubectx kind-k8ssandra-3
+
+helm install cert-manager jetstack/cert-manager --namespace cert-manager \
+     --create-namespace --set installCRDs=true
+```
+
+**Output:**
+
+```bash
+NAME: cert-manager
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
+
+### Install K8ssandra Operator in the control-plane
+
+In this example, of the four clusters we've created, we'll use `kind-k8ssandra-0` as our control-plane.
+
+```bash
+kubectx kind-k8ssandra-0
+
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator \
+     --create-namespace
+```
+
+### Install K8ssandra Operator in the data-planes
+
+In this example, we'll use the three other clusters as data-planes.
+
+```bash
+kubectx kind-k8ssandra-1
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator \
+ --create-namespace
+
+kubectx kind-k8ssandra-2
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator \
+ --create-namespace
+
+kubectx kind-k8ssandra-3
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator \
+ --create-namespace
+```
+
+### Verify control-plane configuration
+
+```
+kubectx kind-k8ssandra-0
+
+kubectl -n k8ssandra-operator get deployment k8ssandra-operator \
+ -o jsonpath='{.spec.template.spec.containers[0].env[?(@.name=="K8SSANDRA_CONTROL_PLANE")].value}'
+```
+
+**Output:**
+
+```bash
+true
+```
+
+### Verify data-plane configuration
+
+We could test for `K8SSANDRA_CONTROL_PLANE`, which for each of the three clusters in our example serving as data-planes, should return `false`. Just one example:
+
+```
+kubectx kind-k8ssandra-1
+
+kubectl -n k8ssandra-operator get deployment k8ssandra-operator \
+ -o jsonpath='{.spec.template.spec.containers[0].env[?(@.name=="K8SSANDRA_CONTROL_PLANE")].value}'
+```
+
+**Output:**
+
+```bash
+false
+```
+
+### Generate and install ClientConfigs
+
+[create-clientconfig.sh](https://github.com/k8ssandra/k8ssandra-operator/blob/main/scripts/create-clientconfig.sh) lives in the k8ssandra-operator repo. It is used to configure access to remote clusters. 
```

Comment:
This can probably be done outside of this doc here, but we'll want to overview somewhere what ClientConfigs are and how they are created and used.  They are key to the communication across clusters.

I just created this issue, we'll get someone to provide some intro documentation for the resource ASAP so you can use it:  https://github.com/k8ssandra/k8ssandra-operator/issues/427

Reply:
Ok I've noted that separate doc need. Will resolve this comment, for now.